gateway/tracing.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/sirupsen/logrus"
```

Comment:
@andrei-tyk please remove this file from commit, as this is comming from one of my commits that wasn't in scope for this PR